coops-ui/src/components/AnnualReport/AGMDate.vue

```diff
@@ -162,7 +160,7 @@ export default class AGMDate extends Mixins(DateMixin) {
    * The current year.
    */
   private get currentYear (): number {
-    return this.currentDate ? +this.currentDate.substring(0, 4) : null
+    return this.currentDate ? +this.currentDate.substring(0, 4) : 0
```

Comment:
FYI - this change makes no difference (except to return type) as this getter is only used in comparisons, and null and 0 have the same result.
Also, currentDate should always be set, so checking it is for code safety anyway.